Upgrade node version required

The version of node we are using (6.5.0) no longer works for decrediton since one of the 2nd level deps we use is now broken on it.
```
npm ERR! Linux 4.4.0-59-generic
npm ERR! argv "/home/jcv/.nvm/versions/node/v6.5.0/bin/node" "/home/jcv/.nvm/ver
sions/node/v6.5.0/bin/npm" "install"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE

npm ERR! electron-chromedriver@1.5.0 install: `node ./download-chromedriver.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-chromedriver@1.5.0 install script 'node ./downlo
ad-chromedriver.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-chromedriver
 package,
```

The latest 'stable' version of node, 6.9.5 seems to work on all of our platforms.

--- a/README.md
+++ b/README.md
@@ -47,8 +47,7 @@ From there follow the on screen instructions to setup your wallet.
 ## Developing
 
 Due to potential compatibility issues, for now, all work should be
-done with node v6.5.0 and electron 1.4.6.  It is likely we can relax
-this once things have settled down.  The recommended way to install
+done with node v6.9.5 and electron 1.4.6.  The recommended way to install
 node is using nvm.
 
 This has primarily been tested on Linux at the moment although OSX
@@ -57,9 +56,9 @@ should work similarly.
 ``` bash
 git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`
 . ~/.nvm/nvm.sh
-nvm install v6.5.0
-nvm use v6.5.0
-nvm alias default v6.5.0
+nvm install v6.9.5
+nvm use v6.9.5
+nvm alias default v6.9.5
 npm install -g npm
 cd
 ```
@@ -117,7 +116,7 @@ you are using msys2 with various development tools (copilers, make,
 ect) all installed.
 
 Install node from the official package https://nodejs.org/en/download/
-and add it to your msys2 path.
+and add it to your msys2 path.  You must install the same version of node as required for Linux and OSX (6.9.5).
 
 Install openssl from the following site:
 https://slproweb.com/products/Win32OpenSSL.html

